CHANGELOG.md

```diff
@@ -3,8 +3,10 @@
 * [#209](https://github.com/slack-ruby/slack-ruby-bot/pull/209): Allow `respond_to_slack_message` and `respond_to_slack_messages` without arguments - [@dblock](https://github.com/dblock).
 * [#216](https://github.com/slack-ruby/slack-ruby-bot/pull/216): Added `start_typing` RSpec matcher - [@dblock](https://github.com/dblock).
 * [#214](https://github.com/slack-ruby/slack-ruby-bot/pull/214): Add passenger deployment documentation - [@cybercrediators](https://github.com/cybercrediators).
+* [#220](https://github.com/slack-ruby/slack-ruby-bot/pull/220): examples/market now pulls from IEX instead of defunct yahoo service - [@corprew](https://github.com/corprew).
```

Comment:
It wants this to be a capital Examples, so maybe "Updated `examples/market` to use IEX cloud API" ?

Reply:
It would be way easier to contribute to this project if prbot/danger produced useful output.

Reply:
I'm not saying that something like this isn't useful or a good engineering practice, especially for lowering the cost to maintainers, but i remember also playing 'guess the lexical features that the bot cares about' a while ago when i tried to contribute to a different slack-ruby repo.

Reply:
Please feel free to improve the bot, https://github.com/dblock/danger-changelog/issues